support/ebpf/dotnet_tracer.ebpf.c

```diff
@@ -102,12 +102,26 @@ dotnet_find_code_start(PerCPURecord *record, DotnetProcInfo *vi, u64 pc, u64 *co
     // This is unrolled several times, so it needs to be minimal in size.
     // And currently this is the major limit for DOTNET_FRAMES_PER_PROGRAM.
     int orig_pos = pos;
-#pragma unroll 256
-    for (int i = 0; i < map_elements - 2; i++) {
-      if (val != 0) {
-        break;
+    if (val == 0) {
+      // pos has already been decremented twice, pos = map_elements - 2.
+      // The `-1` indicates moving to the next u64.
+      int pos64 = (map_elements - 2) / 2 - 1;
+      u64 val64 = scratch->map64[pos64];
+#pragma unroll((128 - 2) / 2 - 1) // be same with pos64
```

Comment:
Why not leave it like:
```suggestion
#pragma unroll 256
```

The unroll works so that it gives the maximum unroll amount to do. So this is equivalent. Not specifying number will just some internal compiler limit for the unroll count. Just saying a large number here should be good.